### PR TITLE
Sensor documentation. WIP

### PR DESCRIPTION
**Pull request by CleoQc**
document how to use the light sensors, tutorial is "Chase the Light"

Changes to the API:
read_sensor(_, BOTH) now returns LEFT first, then RIGHT, to be consistent with the rest of the user-facing code. (LEFT is 0, RIGHT is 1). 

Also the constants for the I2C command to read sensors have been changed. the GET_ has been dropped to make it easier to type.  As there is no SET_ (which wouldn't make sense) it's impossible to get confused. 

ETA: line follower documentation and tutorial is in

### CODE REVIEW

File: docs/source/sensors.rst [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+floor. Many cats do that when you shine a flashlight in front of them, they will 
+try to hunt the light spot. When running this code, you will be able to guide 
+your GiggleBot by using a flashlight. 
+
+This is how to use this project:
+
+#. start GiggleBot and wait for sleepy face to appear on the Micro:Bit,
+#. press button A to start the Chase the Light game (heart will replace the sleepy face),
+#. chase the light as long as you want,
+#. press button B to stop the GiggleBot and display sleepy face again.
```
Comment (RobertLucian) [inline docs/source/sensors.rst]: In this short tutorial called _Chase the Light_, in each numbered list, can we capitalize the first letter and add a full stop at the end of each subpoint (or at least a semicolon)?

File: docs/source/sensors.rst [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+.. code:
+   left, right = read_sensor(LINE_SENSOR, BOTH)
+
+The lower the number, the darker it is reading. Values can go from 0 to 1023 
+and depend a lot on your environment. If you want to write a line follower 
+robot, it is best to take a few readings first, to get a good idea of what
+numbers will represent a black line, and what numbers represent a white line.
+
+Calibrating the line follower
```
Comment (RobertLucian) [inline docs/source/sensors.rst]: Generally, we capitalize all words in a title except the connectors. What do you say?

File: gigglebot_with_docs.py [hunk 3]
```diff
@@ -207,15 +207,22 @@ def read_sensor(which_sensor, which_side):
     """
     Reads the GiggleBot onboard sensors, light or line sensors.
 
-    :param int which_sensor: reads the light sensors GET_LIGHT_SENSORS (6), or the line sensors GET_LINE_SENSORS (5). Values are from 0 to 1023.
+    :param int which_sensor: reads the light sensors LIGHT_SENSOR (6), or the line sensors LINE_SENSOR (5). Values are from 0 to 1023.
     :param int which_side: reads LEFT (0), RIGHT (1), or BOTH (2) sensors. When reading both sensors, an array will be returned.
 
-    :returns: either an integer or an array of integers.
+    :returns: either an integer or an array of integers (left, then right)
```
Comment (RobertLucian) [inline gigglebot_with_docs.py]: It's not the only place in the API, I'm just pointing out the pattern. Can we capitalize the first letter and a full stop at the end of each sentence for each parameter and return section?

File: docs/source/sensors.rst (hunk 2 above)
Comment (CleoQc) [reply docs/source/sensors.rst]: It goes against everything I've ever learned about languages. But English... Silly English...

File: docs/source/sensors.rst (hunk 1 above)
Comment (CleoQc) [reply docs/source/sensors.rst]: Against my better wishes, I shall do so.  #EnglishIsAMonster

File: docs/source/sensors.rst (hunk 2 above)
Comment (RobertLucian) [reply docs/source/sensors.rst]: Yeah, it's English. In my language it's the exact opposite: you do it with a lower case and generally a semicolon at the end.

**Pull request merged**